src/objects.js

```diff
@@ -1,41 +1,73 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+	name: name,
+	age: age,
+}
+return person
 };
 
 const getName = object => {
-  // your code here
+  return object.name
 };
 
 const getProperty = (property, object) => {
-  // your code here
+	const personDetails = object 
+  	return personDetails.age  
+
+/*I have no idea how the above can work but {const person = object 
+    return person.property} doesnt work? it should just substitute "age" into property but I get undefined each time?? I'm sure i'm just missing something obvious! */ 
```

Comment:
That's one of the limitations of using dot notation - you must pass the name of a property that exists, it can never be dynamic. For this you'll have to use the square notation:

return personDetails[property]

This way, whatever property equals to, is replaced in there.